src/__tests__/cart.test.js

```diff
@@ -0,0 +1,162 @@
+const supertest = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const Dish = require('../models/dish');
+const Cart = require('../models/cart');
+const { User } = require('../models/user');
+
+// Mock the getUserID function
+let customerToken;
+let customerId;
+const req = supertest(app);
+let dishId;
+const dishes = [
+  {
+    name: 'Spaghetti Carbonara',
+    chefId: customerId,
+    description:
+      'Classic Italian pasta dish with eggs, cheese, pancetta, and black pepper.',
+    image: 'spaghetti-carbonara.jpg',
+    ingredients: [
+      'Spaghetti',
+      'Eggs',
+      'Pancetta',
+      'Parmesan Cheese',
+      'Black Pepper',
+    ],
+    price: 12.99,
+  },
+];
+
+const customerUser = {
+  firstName: 'cartCustomer',
+  lastName: 'cartCustomer',
+  password: 'cartCustomer%123',
+  name: 'cartCustomerUser',
+  email: 'cartCustomer@example.com',
+  role: 'customer',
+};
+
+beforeAll(async () => {
+  await db.connectToMongo();
+  const res = await req.post('/api/auth/signup').send(customerUser);
+  customerId = res.body.data._id;
+  [customerToken] = res.headers['set-cookie'][0].split(';');
+  // const dish = new Dish(dishes[0]);
+  // await dish.save();
+  dishId = customerId;
+});
+
+afterAll(async () => {
+  await User.deleteMany({
+    email: customerUser.email,
+  });
+  await Dish.deleteMany({});
+  await Cart.deleteMany({});
+  await db.closeDatabase();
+});
+describe('Cart Endpoints', () => {
+  describe('post /api/cart', () => {
+    it('creates a new cart and return it', async () => {
```

Comment:
same sentence is duplicated in all the tests
ex: creates a new cart and return it